internal/pkg/env/map.go

```diff
@@ -33,13 +34,11 @@ func FromMap(data map[string]string) *Map {
 
 func FromOs() (*Map, error) {
 	m := Empty()
-	envs, err := godotenv.Unmarshal(strings.Join(os.Environ(), "\n"))
-	if err != nil {
-		return nil, err
-	}
-
-	for k, v := range envs {
-		m.Set(k, v)
+	for _, pair := range os.Environ() {
```

Comment:
Povodny kod na Win nefungoval dobre.
- `os.Environ` vrati `[]string` , kde kazda polozka je `key=value`, teda kluc a hodnota premennej prostredia
- v Go sa nedaju ziskat `envs` priamo ako `map`, musi sa parsovat vystup `os.Environ`, ak ich chceme vsetky, ... da sa ziskat iba jedna hodnota priamo, a to cez `os.Getenv`
- Na Win vrato `os.Environ` prazdnu env premennu bez `=` a spadne to na parsovani cez `godotenv`.
- Preto to parsujeme rucne, a ak sa nenajde `=`, tak hodnotu proste preskocime.

Reply:
Na `Linuxe` a MacOs` su tie implementacie rovnocenne.